### PR TITLE
document namespace name lenght limitation

### PR DESCRIPTION
**Pull request by sparkoo**
Signed-off-by: Michal Vala <mvala@redhat.com>

> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.

### What does this PR do?
Document namespace name length limitation

### What issues does this PR fix or reference?
partially fixes the https://github.com/eclipse/che/issues/18146

### PR Checklist

As the author of this Pull Request I made sure that:

- [ ] `vale` has been run successfully against the PR branch
- [ ] Link checker has been run successfully against the PR branch
- [ ] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [ ] Changed article references are updated where they are used (or a redirect has been set up on the docs side):
    - [ ] Dashboard [branding.json](https://github.com/eclipse/che-dashboard/blob/master/src/components/branding/branding.json)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

File: modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc [hunk 1]
```diff
@@ -12,6 +12,15 @@ WARNING: `CHE_INFRA_KUBERNETES_NAMESPACE` and `CHE_INFRA_OPENSHIFT_PROJECT` are
 
 WARNING: By default, only one workspace in the same {orch-namespace} can be running at one time. See xref:running-more-than-one-workspace-at-a-time.adoc[].
 
+[WARNING]
+====
+By {kubernetes} definition, the maximum length of namespace name is limited to 63 characters, including the evaluated placeholders.
```
Comment (metlos) [inline modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc]: The name also needs to be a valid DNS name.
Also, I would maybe rephrase this to slightly simplify the language (IMHO):

{kubernetes} limits the length of a namespace name to 63 characters (this includes the evaluated placeholders). Additionally, the names (after placeholder evaluation) must be valid DNS names.

File: modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc [hunk 2]
```diff
@@ -12,6 +12,15 @@ WARNING: `CHE_INFRA_KUBERNETES_NAMESPACE` and `CHE_INFRA_OPENSHIFT_PROJECT` are
...
+
+On OpenShift with multi-host server exposure strategy, the length is further limited to 49 characters.
+
+Be aware that `_<userid>_` placeholder is evaluated into 36 character long UUID string.
```
Comment (metlos) [inline modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc]: Be aware that the \`\_\<userid\>\_\` placeholder is evaluated into a 36 character long UUID string.

File: modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc (hunk 1 above)
Comment (sparkoo) [reply modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc]: much better, updated

File: modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc (hunk 2 above)
Comment (sparkoo) [reply modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc]: thanks, updated

**Comment by themr0c** [comment]
Do we have an idea which milestone we are targetting?

**Comment by sparkoo** [comment]
> Do we have an idea which milestone we are targetting?

It's related to feature limitation that is already in for many releases. So sooner the better... 7.21 is as soon as we can get :)

**Review by yhontyk — APPROVED** [review]
